notebooks/community/managed_notebooks/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,1044 @@
+{
```

Comment:
<p>Consider linking to this <a href="https://cloud.google.com/vertex-ai/docs/datasets/prepare-text#sentiment-analysis" target="_blank">documentation</a> for more info.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>resolved</p>